tests/reconfig/scripts/create-resources-gw-last.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+num_namespaces=$1
+
+# Create namespaces
+for ((i=1; i<=$num_namespaces; i++)); do
+    namespace_name="namespace$i"
+    kubectl create namespace "$namespace_name"
+done
+
+# Create single instance resources
+kubectl create -f certificate-ns-and-cafe-secret.yaml
+kubectl create -f reference-grant.yaml
+
+# Create backend service and apps
+for ((i=1; i<=$num_namespaces; i++)); do
+    namespace_name="namespace$i"
+    sed -e "s/coffee/coffee${namespace_name}/g" -e "s/tea/tea${namespace_name}/g" cafe.yaml | kubectl apply -n "$namespace_name" -f -
+done
+
+# Create routes
+for ((i=1; i<=$num_namespaces; i++)); do
+    namespace_name="namespace$i"
+    sed -e "s/coffee/coffee${namespace_name}/g" -e "s/tea/tea${namespace_name}/g" cafe-routes.yaml | kubectl apply -n "$namespace_name" -f -
+done
+
+# Wait for apps to be ready
+sleep 60
```

Comment:
would it make sense to run `kubectl wait` against created deployments instead of a sleep?

Reply:
`wait` is a bit tricky with multiple resources (I don't want to have to wait per resource, but I couldn't get waiting for resources with different names across different namespaces) so I think sleep is probably the easiest for now